frontend/packages/pipelines-plugin/src/components/pipelineruns/detail-page-tabs/TaskRuns.tsx

```diff
@@ -9,7 +9,7 @@ interface TaskRunsProps {
 const TaskRuns: React.FC<TaskRunsProps> = ({ obj }) => (
   <TaskRunsListPage
     showTitle={false}
-    selector={{ 'tekton.dev/pipelineRun': obj.metadata.name }}
+    selector={{ matchLabels: { 'tekton.dev/pipelineRun': obj.metadata.name } }}
```

Comment:
same here, not a valid `selector`

Reply:
> same here, not a valid `selector`

cc @andrewballantyne

Reply:
Interesting... not sure how it worked without a valid selector 😕